apps/voting/app/src/components/VotePanelContent.js

```diff
@@ -205,20 +205,18 @@ class VotePanelContent extends React.Component {
           </div>
         </SidePanelSplit>
         <Part>
-          {metadata && (
-            <React.Fragment>
-              <h2>
-                <Label>Question</Label>
-              </h2>
-              <Question>{metadata}</Question>
-            </React.Fragment>
-          )}
-          {description && (
+          <React.Fragment>
+            <h2>
+              <Label>Question</Label>
+            </h2>
+            <Question>{questionNode}</Question>
```

Comment:
Technically this could still be empty... but it's probably better to still have the "Question" label rather than removing it?